packages/gatsby-theme-aio/gatsby-config.js

```diff
@@ -154,4 +156,14 @@ module.exports = {
       },
     },
   ],
+  developMiddleware: app => {
+    app.use(
+      "/console/api",
+      createProxyMiddleware({
+        target: "https://developer-stage.adobe.com",
+        secure: false,
+        changeOrigin: true,
+      })
+    )
+  },
```

Comment:
Do we need this? Since dev site is hosted on the same domain as dev console, ideally this should only be needed for local development 🤔